recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
oh whoops did not notice Victor's changed affiliation. congrats then, and sorry for the cc if you no longer work on ampl